Sources/WeatherSDK/WeatherClient.swift

```diff
@@ -3,25 +3,17 @@
 import ClientRuntime
 import Foundation
 import Logging
+import SmithyJSON
+import SmithyReadWrite
```

Comment:
This client, and all the other files in `Sources/WeatherSDK` and `Tests/WeatherSDKTests`, have been updated with new codegen.  The WeatherSDK builds, and its associated tests all pass.

You can look at these files if you want to get an idea of what the new JSON code looks like compared to old.